script/Ciel/Day 10/10_01_CIEL10_4_2C.txt

```diff
@@ -88,38 +90,46 @@ I'm no expert, but is it because it affects the way you fight?"
 [sha:0955a4ac102e72ca39d8ac5d83ed80ddd893bbfb]{
 -- Page 223, Offset 27678.
 -- あれはですね、探索用と殲滅用に分けたもの、と思ってください」
-Well, think of it like one being for investigation and the other for eradication."
+I guess you could say that one of them is for investigation, and the other is for eradication."
 }
 [sha:859f7cc027487f9776233840d032ad6dac3035d5]{
 -- Page 224, Offset 27679.
 -- 「修道服は汎用のスタイルです。
-"The nun outfit is my general-purpose style.
+"The nun's habit is my general purpose outfit.
```

Comment:
Yeah, I kept it because it's pretty clear he's saying style, but it isn't really the greatest word for the rest of the file. I'd prefer to just redirect, which I guess can be made more obvious if you think the continuity isn't clear. Something like
> Ah, so by style you meant my clothing....

for offset 27677.

Reply:
It's certainly better that way, but it feels like a bit of a half-assed way of doing things, I guess. Once you decide to translate 'style' as 'style', and then stop doing it halfway through, there is a certain level of dissonance that I bet most readers would feel.

Perhaps using the words 'clothing style' or 'fashion style' could somewhat reconcile this issue.

Reply:
I'd be surprised if _most_ people felt that. I personally think that awkward use of 'style' in EN is more likely to stick out to the average reader, esp in sentences like
> 「遠野くんが見たあのスタイルは、甲冑が剥がれた後のインナーなんです。
> "The outfit you saw me in back then was actually the undergarments... 

where it's talking about a pretty specific set of clothes.